rideshare/Controllers/RideOfferingsViewController.swift

```diff
@@ -12,14 +12,26 @@ class RideOfferingsViewController: UIViewController, UITableViewDelegate, UITabl
     
     var rides = [PFObject]()
     var selectedPost: PFObject!
-
+    
+    @IBOutlet weak var FilterButton: UIButton!
```

Comment:
these should probably be named `filterButton` and `sortButton` so it's clear that they're not a class or something! that said I know we discussed briefly removing the ability to filter/sort for the demo. do we want to keep the buttons in for this saturday's demo day? would appreciate your opinion as well @tahaafzal5

Reply:
We can keep them (to show as a future feature in the demo) or get rid of them - I don't mind anyway we choose to go.